apps/e2e/cypress/e2e/admin/pools.cy.ts

```diff
@@ -229,33 +229,37 @@ describe("Pools", () => {
     cy.wait("@gqlallPoolsQuery");
 
     // Navigate to edit pool page
-    cy.findByRole("button", { name: /show 10/i }).click();
-    cy.get("[role=menuitemradio]").contains(/50/i).click();
+    cy.findByRole("button", { name: /show 10/i })
+      .click()
+      .then(() => {});
+    cy.get("[role=menuitemradio]").contains(/50/i).click({ force: true });
```

Comment:
Ooh, these force clicks make my spidey sense tingle.  Were you finding something wasn't always ready here?  I'd rather see an explicit wait on something instead.

Reply:
Huh, I'm not sure why but it was failing without this (I was just following the tests radix wrote to try and fix it). But just removing these seem to work fine locally 🤷‍♀️ 

Removed in [remove workarounds from dropdown menu](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8246/commits/26fcbea8c7f4475b8357a2139d2e56b9894db397)